extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/videos/GoogleVideosImporterTest.java

```diff
@@ -85,6 +91,15 @@ public class GoogleVideosImporterTest {
   private PhotosLibraryClient client;
   private UUID jobId;
 
+  class TestStatusCode implements StatusCode {
```

Comment:
I couldn't find an obvious implementation of the `StatusCode` interface in the codebase - happy to swap this out with something more sensible if it exists